State correct talk room (#61)
kitctf/www: _posts/2024-04-11-intro.md

---

Starting on the 25th of April, we will be holding introductory talks for the main categories of Capture the Flag.
We meet on Thursdays at 7pm in the CS building [50.34](https://www.kit.edu/campusplan/), **room -101**.

> **Attention:** This is not customary meeting place but the lecture room, for space reaons.

Reviewer: ```suggestion
**Attention:** This is not our customary meeting place. We meet there for space reasons.
```
